src/sentry/utils/committers.py

```diff
@@ -58,7 +58,7 @@ def score_path_match_length(path_a: str, path_b: str) -> int:
     return score
 
 
-def get_frame_paths(event: Event) -> Union[Any, Sequence[Any]]:
+def get_frame_paths(event: Union[Event, GroupEvent]) -> Union[Any, Sequence[Any]]:
```

Comment:
this func and the one changed in sdk_crash_detection.py only reference `event.data` so i believe it is safe to expect either Event or GroupEvent